### PR TITLE
ISV-3212 Signature Verification Pipeline fails after OSD PROD upgrade

### PR DESCRIPTION
**Pull request by mgreczi**
ISV-3212

### CODE REVIEW

**Comment by Allda** [comment]
@mgreczi Can you please provide a summary of this change and how exactly it helps with the issue?

**Comment by mporrato** [comment]
@Allda the problem is that the `podman image trust set` command tries to update `/etc/containers/policy.json` but UID 1000 cannot change that file.
The idea was to pre-populate policy.json with the correct public key and drop the image trust command entirely.

File: ansible/roles/index-signature-verification/files/tasks/verify-index-signatures.yml [hunk 1]
```diff
@@ -36,6 +39,9 @@ spec:
     - name: keys
       configMap:
         name: "$(params.keys_config_map)"
+    - name: policies
```
Comment (Allda) [inline ansible/roles/index-signature-verification/files/tasks/verify-index-signatures.yml:42]: How does it know where to mount the volume? Other volumes are mounted to a specific location, but this one isn't.

Comment (mgreczi) [reply ansible/roles/index-signature-verification/files/tasks/verify-index-signatures.yml]: Signature verification now works. I update this Pull Request with the working changes.

**Review by mporrato — APPROVED** [review]

**Review by Allda — APPROVED** [review]

**Pull request merged**